content/blog/release-v0.14.7.md

```diff
@@ -0,0 +1,65 @@
++++
+title = "SILE v0.14.7 Is Released"
+date = "2022-12-30"
+
+[extra]
+post_author = "Caleb Maclennan"
+post_gravatar = "11cdaff4c6f9b290db40f69d3b20caf1"
++++
+
+Version v0.14.7 of SILE has been released and is [available for download][release]!
+See the included [CHANGELOG.md][changelog] or review the [commit history][commits] for more explicit details.
+
+### Summary of Improvements
+
+Some of the changes that came out in v0.14.6 were … less than optimal.
+The build system changes were for a good purpose, but made life a bit difficult for some distro packagers.
+I also managed to introduce a parsing regression.
+It also came to light that a previous API deprecation didn't come with the usual warning and shim for transitional support.
+
+This release is notable for distribution packagers for two reasons.
+First, there is now an easier way to skip the font dependency check at build time.
+Second, this release is the first time using LuaJIT is at 100% parity with regular Lua.
+If your distro has LuaJIT available there is now little reason not to make the switch.¹
+The speed increase is substantial, especially for large documents.²
+
+While I was tinkering with smoothing out the build and fixing regressions another notable fix landed.
+
+[Khaled Hosny](https://github.com/khaledhosny) tackled a long standing issue with the font loader.
+The `opsz` axis can now be used as expected so fonts with OpentType FeatureVariations are usable in SILE.
```

Comment:
As long as it does not involve any glyph shapes variation, so it is only for limited use until the larger variable fonts change lands.

Reply:
Yeah, if Khaled's okay with it I'd actually probably hold off on the mention until next release because it's a lot more impressive to say we did it all than "we did one small thing" "ok now we did the rest"

Reply:
Yes, I’d rather keep it for the next release announcement indeed.

Reply:
Fair enough since you both feel that way. I'll adjust, thanks for the feedback.